contracts/contracts/core/BidderRegistry.sol

```diff
@@ -192,8 +201,11 @@ contract BidderRegistry is
         // Transfer funds back to the bidder wallet
         uint256 fundsToReturn = bidState.bidAmt - decayedAmt;
         if (fundsToReturn > 0) {
-            (bool success, ) = payable(bidState.bidder).call{value: (fundsToReturn)}("");
-            require(success, TransferToBidderFailed(bidState.bidder, fundsToReturn));
+            if (!payable(bidState.bidder).send(fundsToReturn)) {
+                // edge case, when bidder is rejecting transfer
+                emit TransferToBidderFailed(bidState.bidder, fundsToReturn);
+                lockedFunds[bidState.bidder][windowToSettle] += fundsToReturn;
```

Comment:
Bidder could still eventually retrieve these funds in `withdrawFromWindows` right?